test/unit/template_importer_test.rb

```diff
@@ -225,18 +225,57 @@ def self.import!(name, text, _metadata)
       end
     end
 
+    test 'should import files from git' do
+      setup_settings
+      imp = ForemanTemplates::TemplateImporter.new
```

Comment:
Oh I see, you're not using `@importer` - but that means you're cloning from the real community-templates repo instead of from the fixtures....